packages/google_maps_flutter/lib/src/marker.dart

```diff
@@ -16,6 +16,8 @@ class Marker {
   Marker(this._id, this._options);
 
   final String _id;
+  String get id => _id;
```

Comment:
nit: add a Dartdoc comment, it should probably say that this is a unique identifier of the marker, and that no assumptions can be made regarding the identifier's format.

Note that on iOS we just generate some unique string as [GMSMarker](https://developers.google.com/maps/documentation/ios-sdk/reference/interface_g_m_s_marker) has no id field, so I would rather not make any commitments about its format (hence the comment )

Reply:
Thanks in!